lib/apimatic-core/utilities/logger_helper.rb

```diff
@@ -23,27 +23,41 @@ def self.get_content_length(headers)
       headers.find { |key, _| key.downcase == CONTENT_LENGTH_HEADER }&.last || ''
     end
 
-    def self.extract_headers_to_log(headers_to_include, headers_to_exclude, headers_to_unmask, headers)
+    def self.extract_headers_to_log(http_object, headers)
```

Comment:
Nits, Blocker: what does `http_object` mean here? It should be `http_logging_configuration`/`http_logging_config`.